volttron/platform/control.py

```diff
@@ -746,6 +748,10 @@ def list_agents_rpc(opts):
             try:
                 peer_method_metadata[peer][method] = conn.server.vip.rpc.call(
                     peer, f'{method}.inspect').get(timeout=4)
+                authorized_capabilities = conn.server.vip.rpc.call(
```

Comment:
Instead of 2 rpc calls to agent, can we have 1 rpc call in agents auth subsystem that does both the job?

Reply:
It is possible, though with how inspect is used, and since this is a command line check usually only calling one or two methods, I don't think it causes a large drain on resources?